browser/react/components/Login/Signup.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+
+export default (props) => {
+  console.log('PROPS', props);
+  return (
+    <div style={props.styles.signUpContainer}>
+      <form onSubmit={props.signup}>
+        <div className="form-group">
```

Comment:
Same comment as above^...I'm going to stop commenting on these now but more down below!